components/phone-lottie.tsx

```
@@ -0,0 +1,377 @@
+    ];
+  }
+
+  function rgbToHsl(r: number, g: number, b: number): [number, number, number] {
+    r /= 255;
+    g /= 255;
+    b /= 255;
+    const max = Math.max(r, g, b),
+      min = Math.min(r, g, b);
+    let h = 0,
+      s = 0,
+      l = (max + min) / 2;
```

ESLint

[error] 'l' is never reassigned. Use 'const' instead.

(prefer-const)